attest/verifier/build.rs

```diff
@@ -290,4 +86,185 @@ fn main() {
     if sgx.ias_mode() == IasMode::Development {
         cargo_emit::rustc_cfg!("feature=\"ias-dev\"");
     }
+
+    if should_generating_sim_files(&data_path) {
+        generate_sim_files(&data_path);
+    }
+}
+
+const ROOT_ANCHOR_FILENAME: &str = "root_anchor.pem";
+const SIGNER_KEY_FILENAME: &str = "signer.key";
+const CHAIN_FILENAME: &str = "chain.pem";
+
+const GENERATED_FILENAMES: &[&str] = &[ROOT_ANCHOR_FILENAME, SIGNER_KEY_FILENAME, CHAIN_FILENAME];
+
+fn generate_sim_files(data_path: impl AsRef<Path>) {
+    let data_path = data_path.as_ref();
+
+    let root_anchor_path = data_path.join(ROOT_ANCHOR_FILENAME);
+    let signer_key_path = data_path.join(SIGNER_KEY_FILENAME);
+    let chain_path = data_path.join(CHAIN_FILENAME);
+
+    const ROOT_SUBJECT: &str = "C=US,ST=CA,L=Santa Clara,O=Intel Corporation,CN=Simulation Intel SGX Attestation Report Signing CA\0";
+    const SIGNER_SUBJECT: &str = "C=US,ST=CA,L=Santa Clara,O=Intel Corporation,CN=Simulation Intel SGX Attestation Report Signer\0";
+
+    let mut serial: [u8; 1] = [1u8];
+
+    let now = Utc::now();
+    let end_now = now;
+    // Starting 1 hour ago
+    let start_time = now - Duration::hours(1);
+    // Good for 30 days
+    let end_time = end_now + Duration::weeks(1);
```

Comment:
Might be a good time to fix this

Reply:
oh yah, I was going to bump it to 30 days, unless you know the history of 1 week?